repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/dataset.ftl

```diff
@@ -100,14 +100,15 @@
             <#list dataset.files as attachment>
                 <li class="attachment" itemprop="hasPart" itemscope itemtype="http://schema.org/MediaObject">
                     <@externalstorageLink attachment.resource; url>
-                    <a itemprop="contentUrl" title="${attachment.text}" href="${url}" onClick="logGoogleAnalyticsEvent('Download attachment','Data set','${attachment.resource.filename}');"><span itemprop="name">${attachment.text}</span></a>;
+                    <a itemprop="contentUrl" title="${attachment.text}" href="${url}" onClick="logGoogleAnalyticsEvent('Download attachment','Data set','${attachment.resource.filename}');" onKeyUp="return vjsu.onKeyUp(event)"><span itemprop="name">${attachment.text}</span></a>;
+                    <a itemprop="contentUrl" title="${attachment.text}" href="${url}" onClick="logGoogleAnalyticsEvent('Download attachment','Data set','${attachment.resource.filename}');" onKeyUp="return vjsu.onKeyUp(event)"><span itemprop="name">${attachment.text}</span></a>;
```

Comment:
Why is it here twice?

Reply:
Oops, that's my bad, the duplication should be gone now!